metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name"        : "puppetlabs-iis",
-  "version"     : "4.2.1",
+  "version"     : "4.2.2",
```

Comment:
Since we're adding the ability to manage site limits, shouldn't this be a minor version rev instead of a patch?

Reply:
Concurred during team chat, moving to `v4.3.0`